### DISCUSSION TITLE
Why is LobeChat Calling Itself 'Apache 2.0 Licensed' When the ASF Prohibits Using 'Apache' in Modified Licenses?

### DISCUSSION THREAD

**Discussion by limond** [lobehub/lobehub#4196] [Development Question]
First, let me thank you for this great project. I have been using LobeChat since before the license change, and the quality of this project is remarkable. I understand that any organization wants to protect their work in a way they feel is necessary to achieve their goals.

However, I am concerned about the use of the term "Apache License 2.0" for this project when it is clearly not the standard Apache License 2.0. You have added new conditions (e.g., "a commercial license must be obtained from the producer if you want to develop and distribute a derivative work based on LobeChat") that are not in the spirit of the original license.

The Apache Software Foundation (ASF) clearly states in their [license FAQ](https://www.apache.org/foundation/license-faq.html#mod-license):

> Also, you cannot use 'Apache' in the name of the modified license. Names like "Apache License with such-and-such clause", for example, are not acceptable, as they cause confusion.

By continuing to refer to this project as "Apache 2.0 licensed," it creates confusion among developers who might assume that this project is fully compatible with other Apache 2.0 licensed works, which it is not. This could lead to unintended legal and practical complications.

I understand your desire to protect your work, and I believe it is only fair to also respect the work of the ASF by not modifying their license and still calling it "Apache." Would you consider renaming your modified license to something that reflects its unique conditions and does not include "Apache"? This would help maintain clarity and respect the guidelines set forth by the ASF.

**Comment by arvinxx** [comment]
Hi @limond , we have updated it to `LobeHub Community License`. Thanks for your advice!
